package.json

```diff
@@ -11,12 +11,13 @@
     "start": "node public/app.js",
     "clean": "rimraf public",
     "build": "npm run clean && npm run build:ts && node public/esbuild.js",
-    "lint": "npx eslint . --ignore-pattern \"node_modules/*\" --ignore-pattern \"public/*\" --fix",
+    "lint": "npx eslint . --ignore-pattern \"node_modules/*\" --ignore-pattern \"public/*\" --ignore-pattern \"test/**/*.spec.ts\" --fix",
     "dev": "nodemon --watch . --ext js,ts,json,scss --ignore node_modules --ignore public/ --delay 500ms --exec 'npm run build:ts && node public/start-server.js'",
     "deploy": "./scripts/deploy.sh",
     "prepare": "husky",
     "tsc": "tsc --noEmit",
-    "build:ts": "tsc"
+    "build:ts": "tsc",
+    "test": "cross-env TS_NODE_PROJECT=tsconfig.test.json NODE_OPTIONS='--loader ts-node/esm' mocha 'test/**/*.spec.ts' --reporter list"
```

Comment:
Rename this script to `test:unit` to make the purpose of the script clearer and to follow a more structured testing convention. This will allow us to scale testing easily by running `npm run test` as an umbrella command for all test types in future.